### ISSUE TITLE
Crash on start

### ISSUE THREAD

**Issue by graphixillusion** [mattstermiller/koffee#75]
Hi there. I'm using the last 0.5.0 version in Windows 10 1903. The app crash upon start with this errors in the system event log:

```
EVENT ID: 1000 (Origin Application Error)
Name of the app which has generate the error: Koffee.exe, version: 0.5.0.0, timestamp: 0x5dc95e73
Name of the module which has generate the error: KERNELBASE.dll, version: 10.0.18362.449, timestamp: 0xa0c7be3c
Exception Code: 0xe0434352
Offset Error 0x00113572
Process ID which has generate the error: 0x1b00
```
```
EVENT ID: 1026 (Origin .NET Runtime)
Application: Koffee.exe
Framework Version: v4.0.30319
Description: unhandled exception
Exception Info: System.ArgumentException
   in System.IO.FileSystemWatcher..ctor(System.String, System.String)
   in Koffee.PersistFile`1[[System.__Canon, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089]]..ctor(System.String, System.__Canon)
   in Koffee.Program.main(System.String[])
```

**Comment by FoggyFinder** [comment]
Seems like there is no check that directory / file exists

**Comment by mattstermiller** [comment]
Yikes! Sorry about that. Thanks for letting me know!

I should be able to look into this today or tomorrow, but if either of you would like to submit a PR that would be lovely. I will definitely deploy a new release to fix this severe issue once it's fixed.

**Comment by mattstermiller** [comment]
Sorry, I should have mentioned that there is a workaround to the issue: Create a "Koffee" folder in your user profile/AppData/Roaming folder.

**Comment by mattstermiller** [comment]
Apologies, it took me longer to get to this than I wanted, but this has now been fixed and published as version 0.5.1. You can get it on the [releases page](../releases).
